src/Smart.CFA.Pages/README.md

```diff
@@ -0,0 +1,3 @@
+#Client CFA
+
+Client-facing Razor Page web application in charge of rendering the the CFA views
```

Comment:
- Please use actual sentences (i.e. subject, verb, ...) and end with a final dot.
- There is a double "the the".

Reply:
Good catch, will rewrite the readme sentence